package.json

```diff
@@ -215,6 +219,8 @@
   "files": [
     "bin/*",
     "lib/*",
+    "!lib/**/*.ts",
+    "!lib/.gitigore",
```

Comment:
I assume this is a typo

Reply:
No, this was intentional. I put them there so that nothing but the compiled `.js` files would be bundled into the NPM package when published.

Reply:
gitigore? not gitignore?

Reply:
Ohhhhh! Yeah 😅 that is a typo. Fixed.